include/mega/crypto/sodium.h

```diff
@@ -19,7 +19,6 @@
  * program.
  */
 
-#ifdef USE_SODIUM
```

Comment:
It might be good to use ENABLE_CHAT here to save space on apps that don't want to link libsodium.